### PR TITLE
Configure non-expiring tokens

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-3123

/cc @openshift/test-platform

### CODE REVIEW

File: clusters/psi/Makefile [hunk 1]
```diff
@@ -3,11 +3,19 @@ SHELL=/usr/bin/env bash -o errexit
...
+CLUSTER ?= app.ci
+SA_NAMESPACE ?= ci
+API_SERVER_URL ?= "https:\\/\\/api.ci.l2s4.p1.openshiftapps.com:6443"
 
-serviceaccounts = release-schedules-cm-updater sync-rover-groups-updater
+serviceaccounts = sync-rover-groups-updater
 
 $(serviceaccounts):
-	oc --context app.ci sa create-kubeconfig -n ci $@ | sed 's/$@/app.ci/g' > "$(TMPDIR)/sa.$@.app.ci.config"
+ifeq ($(TOTATE), true)
```
Comment (smg247) [inline clusters/psi/Makefile:13]: Is this supposed to be:
```suggestion
ifeq ($(ROTATE), true)
```

**Review by smg247 — CHANGES_REQUESTED** [review]

File: core-services/ci-secret-generator/_config.yaml [hunk 2]
```diff
@@ -191,21 +191,3 @@
     - build02
     service_account:
     - sprint-automation
-- fields:
```
Comment (hongkailiu) [inline core-services/ci-secret-generator/_config.yaml:194]: I will start to refresh the tokens (from `secret/config-updater-token-version-1`) in vault manually after merge.

File: Makefile [hunk 3]
```diff
@@ -355,3 +357,53 @@ update_crt_crd: download_crt_crd $(crds)
...
+	mv $(TMPDIR)/_token.yaml ./hack/_token.yaml 
+.PHONY: increase-token-version
+
+refresh-token-version:
+	grep -r -l "ci.openshift.io/token-version: version-$(pre_token_version)" ./clusters | while read file; do $(sed_cmd) -i "s/version-$(pre_token_version)/version-$(token_version)/g" $${file}; done
+.PHONY: refresh-token-version
+
+DRY_RUN ?= server
+
+expire-token-version:
```
Comment (smg247) [inline Makefile:376]: Is this forthcoming?

File: Makefile [hunk 4]
```diff
@@ -355,3 +357,53 @@ update_crt_crd: download_crt_crd $(crds)
...
+CLUSTER ?= app.ci
+API_SERVER_URL ?= "https://api.ci.l2s4.p1.openshiftapps.com:6443"
+TMPDIR ?= /tmp
+
+config-updater-kubeconfig:
+	$(timeout_cmd) 60 ./clusters/psi/create_kubeconfig.sh "$(TMPDIR)/sa.config-updater.${CLUSTER}.config" ${CLUSTER} $@ ci ${API_SERVER_URL} config-updater-token-version-$(token_version)
+	cat "$(TMPDIR)/sa.config-updater.${CLUSTER}.config"
+.PHONY: config-updater-kubeconfig
+
+secret-config-updater:
```
Comment (smg247) [inline Makefile:397]: This is something that will have to be called out when adding a new cluster to the build farm. I am not sure that `cluster-init` can handle making the change automatically.

File: docs/dptp-triage-sop/token-rotation.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+Then, populate the new version in other places:
+
+> make refresh-token-version
+
+Then create a pull request based on the changes and merge it.
+
+### Generate New Tokens
+
+#### Config-Updater
+The tokens in [dptp/config-updater](https://vault.ci.openshift.org/ui/vault/secrets/kv/show/dptp/config-updater) has to be refreshed manually.
```
Comment (smg247) [inline docs/dptp-triage-sop/token-rotation.md:29]:
```suggestion
The tokens in [dptp/config-updater](https://vault.ci.openshift.org/ui/vault/secrets/kv/show/dptp/config-updater) have to be refreshed manually.
```

**Comment by hongkailiu** [comment]
Updated the tokens in Vault.

**Pull request merged**